probe_scraper/parsers/metrics.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+from glean_parser.parser import parse_metrics
+from pathlib import Path
+
+
+class MetricsParser:
```

Comment:
nit: can we call this something like `GleanMetricsParser` or, at least, mention that's used for glean in the docstring for this class?